tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: add addon to SLES via SCC
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base qw(y2_installbase y2_module_guitest);
+use strict;
+use warnings;
+use testapi;
+use registration;
+use version_utils qw(is_sle is_leap);
+use x11utils 'turn_off_gnome_screensaver';
+use YaST::Module;
+use utils;
+
+sub test_setup {
+    select_console "x11";
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+}
+
+sub run {
+    test_setup;
+    YaST::Module::open(module => 'scc', ui => 'qt');
+    #y2_module_guitest::launch_yast2_module_x11('scc', target_match => [qw(scc-registration packagekit-warning)], maximize_window => 1);
+    save_screenshot;
+    assert_screen("scc-registration", 30);
+    wait_screen_change { send_key 'alt-f10' };
+    save_screenshot;
+    $testapi::distri->get_registration()->register_via_scc({
+            email => get_var('SCC_EMAIL'),
+            reg_code => get_var('SCC_REGCODE')});
+    save_screenshot;
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
+
+    # No libyui support on yast_scc-pkgtoinstall page so we have to use assert-screen
+    assert_screen("yast_scc-pkgtoinstall", 100);
+    send_key_until_needlematch 'yast_scc-installation-summary', 'alt-a', 5, 5;
+    $testapi::distri->get_module_registration_installation_report()->press_next();
+    save_screenshot;
+    assert_screen("generic-desktop", 60);
+    # Check that repos actually work
+    select_console 'root-console';
+    zypper_call 'refresh';
+    zypper_call 'repos --details';
```

Comment:
I don't remember, but how we could see the serial output here (do we need to redirect)? [this](https://openqa.suse.de/tests/9386243#step/addon_products_via_SCC_yast2/28) doesn't help much.